akeyless_gateway/manifest.json

```diff
@@ -0,0 +1,64 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "a71a3b29-5921-4bc9-8a7e-38de5a940ad8",
+  "app_id": "akeyless-gateway",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Track your Akeyless Gateway key metrics.",
+    "title": "Akeyless Gateway",
+    "media": [
+      {
+        "caption": "Akeyless Gateway metrics dashboard",
+        "image_url": "images/akeyless_gw_dashboard.png",
+        "media_type": "image"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Security",
+      "Category::Kubernetes",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
```

Comment:
Can you also please add tags here for submitted/queried data types as described [here](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags)?

Reply:
Done. Added:
"Submitted Data Type::Metrics"